tests/management/tenant/test_model.py

```diff
@@ -243,7 +243,7 @@ def test_bulk_adding_updating_users(self):
 
         self.service.update_users(users)
 
-        self.assertEquals(12, self.tuples.count_tuples())
+        self.assertEquals(18, self.tuples.count_tuples())
```

Comment:
Thanks for pointing that out. I was bootstrapping all tenants even if it has been bootstrapped before. After thinking about it, this is redundant, I refactored the code a bit so it would skip those already bootstrapped.